workflow/scripts/filter_vcf.py

```diff
@@ -217,15 +218,15 @@ def convert_to_expression(expression):
         }
 
         # Extract information about how None values should be handled during filtering
-        regex_na_handling = r"(VEP|FORMAT|INFO):(NA_TRUE|NA_FALSE|NA_ERROR):"
+        regex_na_handling = r"(VEP|FORMAT|INFO|QUAL):(NA_TRUE|NA_FALSE|NA_ERROR):"
         na_handling = re.search(regex_na_handling, expression)
         if na_handling:
             na_handling = na_handling.groups()[1]
             # Remove NA handling information from expression
             expression = re.sub(r"(NA_TRUE|NA_FALSE|NA_ERROR):", '', expression)
         else:
             na_handling = "NA_FALSE"
-        regex_string = "[ ]*(VEP|FORMAT|INFO):([A-Za-z0-9_.]+):*([0-9]*)"
+        regex_string = "[ ]*(VEP|FORMAT|INFO|QUAL):([A-Za-z0-9_.]+):*([0-9]*)"
```

Comment:
we don't need to modify this regex when we have a separate if statement handling QUAL, or?